test/whl-scale-tests/whl-scale-test.md

```diff
@@ -0,0 +1,302 @@
+# Windows Host Logs Scale Test
+
+## Prerequisites

```

Comment:
If the script wont install/download nuget.exe for the user it should be listed here as Prereq